crime-commons-classes/src/test/java/uk/gov/justice/laa/crime/util/DateUtilTest.java

```diff
@@ -101,4 +102,16 @@ void givenNullLocalDate_whenGetLocalDateStringIsInvoked_thenReturnNull() {
     void givenAValidLocalDate_whenGetLocalDateStringIsInvoked_thenReturnCorrectString() {
         assertThat(DateUtil.getLocalDateString(LocalDate.of(2023, 10, 6))).isEqualTo("06-Oct-2023");
     }
+
+    @Test
+    void givenAValidLocalDateTime_whenToZonedDateTimeIsInvoked_thenReturnZonedDateTime() {
+        ZonedDateTime expected = ZonedDateTime.parse("2011-12-03T10:15:30.342+00:00");
+        ZonedDateTime actual = DateUtil.toZonedDateTime(LocalDateTime.parse("2011-12-03T10:15:30.342"));
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    void givenNullLocalDateTime_whenToZonedDateTimeIsInvoked_thenReturnNull() {
+        assertThat(DateUtil.toZonedDateTime(null)).isNull();
```

Comment:
Could you add a test for blank string as Date?

Reply:
I think the blank string test is not appropriate here.